src/sfdt/crossReference.ts

```diff
@@ -0,0 +1,195 @@
+import {isBlockList} from './canUseListCondition';
+import {isBookmark} from './../queryBookmark';
+import get from 'lodash/get';
+import filter from 'lodash/filter';
+import {block as BlockType} from '../../types/sfdt';
+import {processSFDT, processBlockForCrossRef, processBlocks} from './blocksProcess';
+import createSfdt from './numberFromList';
+
+export const doInlinesContain = (inlines, callback) => {
+	const newInlines = filter(inlines, (inline) => {
+		return callback(inline);
+	});
+
+	return newInlines;
+};
+
+const ANCHOR = 'XREFANCHOR:';
+const REF = 'XREF:';
+
+const getAnchorName = (anchorBookmark) => anchorBookmark.split(ANCHOR)[1];
+const getRefName = (refBookMark) => refBookMark.split(REF)[1];
+
+const inlinesWithAnchor = (inlines) =>
+	doInlinesContain(inlines, (inline) => {
+		const name = get(inline, 'name');
+
+		const processedName = name; //&& name.split(':');
```

Comment:
can you please do
`const processedName  = get(inline, 'name');`

* same for all other down the file.